XRTK-Core/Packages/com.xrtk.core/Interfaces/Providers/Controllers/IMixedRealityHandVisualizer.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System;
+using XRTK.Definitions.Utilities;
+
+namespace XRTK.Interfaces.Providers.Controllers
+{
+    /// <summary>
+    /// Hand visualization definition, used to provide access to hand joint objects.
+    /// </summary>
+    public interface IMixedRealityHandVisualizer : IMixedRealityControllerVisualizer
+    {
+        /// <summary>
+        /// Get a game object following the hand joint.
+        /// </summary>
+        [Obsolete("Use HandJointUtils.TryGetJointPose instead of this")]
```

Comment:
This seems already obsolete in the MRTK. Maybe XRTK can update to the new implementation right away.

Reply:
Agreed, as it's new we should not include any obsolete code.